lib/routes/account.js

```diff
@@ -1007,7 +1007,7 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
       config: {
         auth: {
           mode: 'optional',
-          strategy: 'sessionToken'
+          strategy: 'sessionTokenWithDevice'
```

Comment:
FWIW I am +1 to having everything fail, the simpler we can keep our auth-related policies, the less likely we are to have weird bugs and interactions and security problems.